README.md

```diff
@@ -98,144 +98,135 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.79 - February 2024 Update
+### 0.80 - March 2024 Update
 
 In this release, we focused on stability and improvements.
 
 **Highlights**
 
- - New feature: Keyboard Manager allows mapping shortcuts to start applications or opening URIs. Thanks [@jefflord](https://github.com/jefflord)!
- - New feature: Keyboard Manager allows shortcuts with chords. Thanks [@jefflord](https://github.com/jefflord)!
- - Modernized Color Picker with Fluent UX. Thanks [@niels9001](https://github.com/niels9001)!
- - Peek now is able to preview drives. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - File Locksmith has now an entry in the Windows 11 tier 1 context menu.
+ - New feature: Desired State Configuration support, allowing the use of winget configure for PowerToys.
+ - The Windows App SDK dependency was updated to 1.5.1, fixing many underlying UI issues.
+ - Webp/webm files support was added to Peek. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Audio files support was added to Peek. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Automated UI testing for FancyZones Editor was added to CI.
 
 ### General
 
- - Refactored code so that English is used as a fallback language when a localized resource cannot be found.
+ - Added a Quick Access entry to access the flyout from PowerToys' tray icon right click menu. Thanks [@pekvasnovsky](https://github.com/pekvasnovsky)!
+ - Added support for Desired State Configuration in PowerToys, allowing the use of winget configure to configure many settings.
```

Comment:
Maybe adding a link to what DSC is or to more information regarding PowerToys DSC could be helpful

Reply:
Thinking of linking to DSC docs on learn.microsoft.com, after opening that PR

Reply:
Added the link in the Highlights mention of DSC.